Observer/src/observer/Main.java

```diff
@@ -0,0 +1,34 @@
+package observer;
+
+public class Main {
+    public static void main(String[] args) {
+        Channel channel = new Channel();
+
+        Subscriber subscriber_1 = new Subscriber("ABC");
+        Subscriber subscriber_2 = new Subscriber("XYZ");
+        Subscriber subscriber_3 = new Subscriber("QWERTY");
+        Subscriber subscriber_4 = new Subscriber("JOHN");
+        Subscriber subscriber_5 = new Subscriber("MARRY");
+
+
+        channel.subscribeChannel(subscriber_1);
```

Comment:
channel should not subscribe subscribers.
line 20 to 24 are correct but not 14 to 18

Reply:
Every channel can subscribe/unsubscribe the channel. It's like T-Series can subscribe me and I can subscribe T-Series